frontends/open-discussions/webpack.config.shared.js

```diff
@@ -16,8 +15,13 @@ module.exports = {
       ]
     },
     resolve: {
-      modules:    [path.join(__dirname, "static/js"), "node_modules"],
-      extensions: [".js", ".jsx"]
+      extensions: [".js", ".jsx"],
+      fallback:   {
+        assert: require.resolve("assert"),
+        os:     require.resolve("os-browserify"),
+        path:   require.resolve("path-browserify"),
+        url:    require.resolve("url")
```

Comment:
This PR upgrades from webpack 4 to webpack 5. Webpack 5 no longer polyfills node core modules (webpack 4 does). See https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5

We (our are dependencies) are using these, so I added them.